amundsen_rds/models/application.py

```diff
@@ -0,0 +1,35 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from sqlalchemy import BigInteger, Column, ForeignKey, String, Text
+
+from amundsen_rds.models.base import (
+    KEY_LEN, NAME_LEN, PUBLISHED_TAG_LEN, URL_LEN, Base
+)
+
+
+class Application(Base):
```

Comment:
the alembic script will be put under metadata repo?

Reply:
yes, I think I will put alembic under metadata repo and make alembic env point to amundsenrds models.